src/test/regress/expected/multi_explain_0.out

```diff
@@ -940,3 +940,38 @@ Custom Scan (Citus Router)
         ->  Seq Scan on explain_table_570001 explain_table
               Filter: (id = 1)
 ROLLBACK;
+-- test explain with local INSERT ... SELECT
```

Comment:
Shall we have an EXPLAIN test with `CTE`s? Since we're manually changing the query, `EXPLAIN` could ensure that we're doing things correctly.